tests/integration/v3_auth_test.go

```diff
@@ -420,83 +420,6 @@ func TestV3AuthOldRevConcurrent(t *testing.T) {
 	wg.Wait()
 }
 
-func TestV3AuthRestartMember(t *testing.T) {
```

Comment:
It is the same verification as `authTestCacheReload` in `ctl_v3_auth_test.go` and it was added in https://github.com/etcd-io/etcd/pull/14358